pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -209,7 +212,12 @@ func (server *VerifierImpl) SendAttestQuote(ctx *zattest.Context) error {
 		log.Errorf("[ATTEST] Invalid response code")
 		return zattest.ErrControllerReqFailed
 	case attest.ZAttestResponseCode_Z_ATTEST_RESPONSE_CODE_SUCCESS:
-		//XXX Retrieve Storage keys, and integrity token
+		//Retrieve Storage keys, and integrity token
+		iToken := quoteResp.GetIntegrityToken()
+		if iToken == nil {
+			log.Fatalf("[ATTEST] Empty integrity token received")
+		}
+		storeIntegrityToken(iToken)
```

Comment:
Could the received token be an empty string? If so do we want to store it as empty?
If so we could also treat the missing file in read as having the empty string content.

Reply:
> Could the received token be an empty string? If so do we want to store it as empty?
> If so we could also treat the missing file in read as having the empty string content.

Replied above.